modules/shared/src/main/resources/logback-included.xml

```diff
@@ -82,5 +96,12 @@
         <appender-ref ref="GOSSIP" />
         <appender-ref ref="JSON" />
     </logger>
+    <logger name="org.tessellation.dag.l1.http.Routes" level="DEBUG" additivity="false">
+        <appender-ref ref="TRANSACTIONS" />
```

Comment:
Maybe a `JSON` appender as well? If we have centralized logs, we're likely going to use json format.

Reply:
We have a plan for better logging, and I'd like to avoid spamming with these logs right now. Tomorrow we'll call that part